content/systems/filecoin_vm/message/_index.md

```diff
@@ -10,41 +10,139 @@ dashboardTests: 0
 # VM Message - Actor Method Invocation
 
 A message is the unit of communication between two actors, and thus the primitive cause of changes
-in state. A message combines:
+in state. When referring to a Message in the context of Filecoin and unless otherwise defined, we refer to a _Transaction Message_. A message combines:
 
 - a token amount to be transferred from the sender to the receiver, and
-- a method with parameters to be invoked on the receiver (optional).
+- a method with parameters to be invoked on the receiver (optional/where applicable).
 
 Actor code may send additional messages to other actors while processing a received message. 
-Messages are processed synchronously: an actor waits for a sent message to complete before resuming control.
+Messages are processed synchronously, that is, an actor waits for a sent message to complete before resuming control.
 
-The processing of a message consumes units of computation and storage denominated in gas. 
-A message's gas limit provides an upper bound on its computation. The sender of a message pays 
+The processing of a message consumes units of computation and storage, both of which are denominated in gas. 
+A message's _gas limit_ provides an upper bound on the computation required to process it. The sender of a message pays 
 for the gas units consumed by a message's execution (including all nested messages) at a
 gas price they determine. A block producer chooses which messages to include in a block and is
 rewarded according to each message's gas price and consumption, forming a market.
 
 ## Message syntax validation
 
 A syntactically invalid message must not be transmitted, retained in a message pool, or
-included in a block.
+included in a block. Even if an invalid message ends up being sent (e.g., by a malicious player), the libp2p GossipSub propagation protocol will reject the message on the receiving end.
```

Comment:
will -> should? 

How about "if an invalid message is received, a node should not propagate it". An implementer needs to tell their gossipsub what a valid message looks like, we can't just say it'll magically be rejected.